internal/storage/v2/clickhouse/tracestore/reader.go

```diff
@@ -215,3 +216,97 @@ func (r *Reader) GetOperations(
 	}
 	return operations, nil
 }
+
+
+func (r *Reader) FindTraces(
```

Comment:
IMO, name should be like this :

```suggestion
func (r *Reader) GetTraces(
```